kafka-init/src/main/java/io/strimzi/kafka/init/Main.java

```diff
@@ -15,6 +15,14 @@
 public class Main {
     private static final Logger LOGGER = LogManager.getLogger(Main.class);
 
+    /**
+     * Default constructor for the Main class.
+     * The constructor is intentionally left empty.
+     */
+    public Main() {
+        // No initialization required
+    }
```

Comment:
is missing a ctor a Javadoc warning?

Reply:
It's using the default empty constructor and yeah, it's not documented, so Javadoc is throwing a warning.
It's a standard way how to document these kind of things (every public API/class should be documented).

```
[WARNING] /Users/lkral/Work/strimzi-kafka-operator/kafka-init/src/main/java/io/strimzi/kafka/init/Main.java:15: warning: use of default constructor, which does not provide a comment
[WARNING] public class Main {
[WARNING] ^
[WARNING] 7 warning
```

Reply:
I know it was using a default empty ctor but didn't imagine that Javadoc was raising a warning for something not explicitly written by the developer but used at runtime.

Reply:
Yeah it's surprising for me as well. But what we can do right 😄 🤷